lib/document/templates.js

```diff
@@ -247,9 +263,35 @@ class PagedReadyHandler extends Paged.Handler {
     }
 }
 Paged.registerHandlers(PagedReadyHandler);
+
```

Comment:
I think we might want to extract this code to a proper JavaScript file so we can apply linter rules and it will be easier to edit in IDE.
What do you think?

Reply:
done in next revision